var/exp_repo/experiments/laghos/experiment.py

```diff
@@ -0,0 +1,62 @@
+from benchpark.directives import variant
+from benchpark.experiment import Experiment
+
+
+class Laghos(Experiment):
+
+    variant(
+        "experiment",
+        default="example",
+        values=("weak", "strong","example"),
+        description="weak or strong scaling",
+    )
+
+    def compute_applications_section(self):
+        variables = {}
+            
+        if self.spec.satisfies("scaling=weak"):
+            variables["rs"] = ["5","6"]
+            variables["n_nodes"] = ["1","8"]
+        elif self.spec.satisfies("scaling=strong"):
+            variables["n_nodes"] = ["1","8"]
```

Comment:
Instead of increasing nodes for strong scaling let's keep it on one node and increase MPI ranks, 1,8,64. We should lower the rs value to 1 or 2 here as well, depending on how long this takes to solve, these would ideally be less than 5 minutes to solve.